miden-tx/src/tests/kernel_tests/test_tx.rs

```diff
@@ -647,32 +647,33 @@ fn test_load_foreign_account_basic() {
         
         use.kernel::prologue
         use.miden::tx
-        use.miden::account
 
         begin
             exec.prologue::prepare_transaction
 
             # pad the stack for the `execute_foreign_procedure`execution
-            padw padw push.0.0.0
-            # => [pad(11)]
+            padw padw padw push.0.0
+            # => [pad(14)]
 
             # push the index of desired storage item
             push.0
 
-            # get the hash of the `get_item_foreign` account procedure
-            procref.account::get_item_foreign
+            # get the hash of the `get_item` account procedure
+            push.{get_item_foreign_hash}
 
             # push the foreign account id
             push.{account_id}
-            # => [foreign_account_id, FOREIGN_PROC_ROOT, storage_item_index, pad(11)]
+            # => [foreign_account_id, FOREIGN_PROC_ROOT, storage_item_index, pad(14)]
 
             exec.tx::execute_foreign_procedure
             # => [STORAGE_VALUE_1]
 
             # truncate the stack
             exec.sys::truncate_stack
         end
-        "
+        ",
+        // `get_item` procedure has index 7
+        get_item_foreign_hash = foreign_account.code().procedures()[6].mast_root(),
```

Comment:
The comment mentions index `7` but the actual index used is `6` - they should probably be in sync.

But also, I would probably create a small helper functions which encapsulates this logic (e.g., maybe something like `get_root_of_get_item_procedure()`) and I'd also document it pretty heavily.